src/System.Data.SqlClient/src/System/Data/SqlClient/LocalDBAPI.Common.cs

```diff
@@ -36,7 +36,6 @@ private static LocalDBFormatMessageDelegate LocalDBFormatMessage
 
                             if (functionAddr == IntPtr.Zero)
                             {
-                                int hResult = Marshal.GetLastWin32Error();
```

Comment:
GetLastWin32Error is side-effect free, right?

Reply:
Yes